features/startup.feature

```diff
@@ -0,0 +1,29 @@
+Feature: Startup
```

Comment:
But it does say in the Feature description `Naemon should start up if and only if there is a valid configuration`. Do you think that is lacking?